### PR TITLE
corrected the phase 3

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6143
session key is not encrypted (it would be completely illogical), source: https://docs.microsoft.com/en-us/archive/blogs/dubaisec/bitlocker-network-unlock

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
could you please have a look?
Thank you

File: windows/security/information-protection/bitlocker/bitlocker-how-to-enable-network-unlock.md [hunk 1]
```diff
@@ -80,7 +80,9 @@ The server side configuration to enable Network Unlock also requires provisionin
 
 1.  The Windows boot manager detects that a Network Unlock protector exists in the BitLocker configuration.
 2.  The client computer uses its DHCP driver in the UEFI to obtain a valid IPv4 IP address.
-3.  The client computer broadcasts a vendor-specific DHCP request that contains the Network Key (a 256-bit intermediate key) and an AES-256 session key for the reply. Both of these keys are encrypted using the 2048-bit RSA Public Key of the Network Unlock certificate from the WDS server.
+3.  The client computer broadcasts a vendor-specific DHCP request that contains: 
+    1.  Network Key (a 256-bit intermediate key) encrypted using the 2048-bit RSA Public Key of the Network Unlock certificate from the WDS server.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-how-to-enable-network-unlock.md:84]:
```suggestion

-  Network Key (a 256-bit intermediate key) encrypted using the 2048-bit RSA Public Key of the Network Unlock certificate from the WDS server.
```

Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-how-to-enable-network-unlock.md:84]:
```suggestion
    1.  A Network Key (a 256-bit intermediate key) encrypted using the 2048-bit RSA Public Key of the Network Unlock certificate from the WDS server.
```

File: windows/security/information-protection/bitlocker/bitlocker-how-to-enable-network-unlock.md [hunk 2]
```diff
@@ -80,7 +80,9 @@ The server side configuration to enable Network Unlock also requires provisionin
...
+    2.  AES-256 session key for the reply.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-how-to-enable-network-unlock.md:85]:
```suggestion
    2.  An AES-256 session key for the reply.
```

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6143.

Thank you.

cc: @kenwith

**Pull request merged**